### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by mukhammadrizooff**
# Leaderboard: Build with API
- I Read the [Leaderboard API documentation](https://www.notion.so/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
- Created a new game with the name of your choice by using the API.
- I implement the "Refresh" button.
- I implement the form "Submit" button.
- I used `async` and `await` JavaScript features to consume the API.
- I didn’t add any additional design.

### CODE REVIEW

File: src/modules/display.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Vgufs5n18vzBAjzxI4Io/scores';
+
+const scoreDisp = () => document.querySelector('#list-scores');
```
Comment (o-rashad) [inline src/modules/display.js]: **_[OPTIONAL]_** it looks like `scoreDisp` and `scoresDisplay` functions namings are similar, 
Let's try to choose names that are easier for your code reader to understand 
without looking at your code 😉👉

**Review by o-rashad — APPROVED** [review]
Asslam Alaikum @student ,😊

Hi👋🏽, Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 

Happy coding!👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/display.js (hunk 1 above)
Comment (o-rashad) [reply src/modules/display.js]: Try to think about what actually these variables represent. 👍

Comment (mukhammadrizooff) [reply src/modules/display.js]: ### @od-c0d3r  Ooh I didn't pay attention to that, thank you for saying :)
### I'll do it right now!

**Pull request merged**